packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -373,6 +377,51 @@ describe("calcite-list", () => {
     expect(visibleItems.map((item) => item.id)).toEqual(["label-match", "description-match", "value-match"]);
   });
 
+  it("filters initially with matchFields", async () => {
```

Comment:
Do we need a test for user-interaction filtering? Under the hood, both work with `filterText`, so probably not?

Reply:
Probably not necessary.